contracts/contracts/system_contracts/kaiabridge/HolderVerifier.sol

```diff
@@ -98,7 +98,6 @@ contract HolderVerifier is Ownable, ReentrancyGuard, IHolderVerifier {
     function requestProvision(
         bytes calldata publicKey,
         string calldata fnsaAddress,
-        bytes32 messageHash,
```

Comment:
Can we still receive the messageHash arg? To avoid frontend change.

Reply:
Yes, we keep the messageHash arg to avoid frontend changes.
Addressed in https://github.com/kaiachain/kaia/pull/695/commits/43e622a07c8cd9c38a5f213b21d4160ce32e0d65.